docs/api/session.md

```diff
@@ -348,8 +348,14 @@ win.webContents.session.setCertificateVerifyProc((request, callback) => {
 
 * `handler` Function | null
   * `webContents` [WebContents](web-contents.md) - WebContents requesting the permission.  Please note that if the request comes from a subframe you should use `requestingUrl` to check the request origin.
-  * `permission` String - Enum of 'media', 'geolocation', 'notifications', 'midiSysex',
-    'pointerLock', 'fullscreen', 'openExternal'.
+  * `permission` String - Types of permission that can be requested.
+    * `media` - Allows app to access media files from a user's device with the user's consent.
+    * `geolocation` - Allows app to get the user's current location.
+    * `notifications` - Allows app to create notifications and display them in the user's system tray.
+    * `midiSysex` - Allows the usage of sysex messages in the `webmidi` API.
+    * `pointerLock` - Allows the app to directly interpret mouse movements as an input method.
+    * `fullscreen` - Allows application to run in fullscreen mode.
+    * `openExternal` - Allows the app to open specific application based on the url and filetype association.
```

Comment:
```suggestion
    * `openExternal` - Allows the app to open a specific application based on the URL and file type association.
```

Reply:
I'm unclear as to what "file type association" means here. From what I understand from the original PR (#5231), this is meant for custom browser protocols.

Reply:
Not browser protocols but generally schemes that are registered by other applications they can handle with the OS, ex: `mailto`, `ms-settings:` etc, so these are schemes external to your app.